kubernetes/deployment/deployment.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: descheduler
+  namespace: kube-system
+  labels:
+    app: descheduler
+spec:
+    replicas: 1
```

Comment:
the indentation seems off for everything under `spec`, could you double check that? Maybe it's just how github is showing it

Reply:
This is a good catch, didn't realize the indentation was off